docs/README.md

```diff
@@ -7,7 +7,7 @@ Precondition: python3 and pip3 are already installed.
 ### Recommended steps:
 
 1. Install virtualenv - this allows you to create a python sandbox for your project  
-`pip3 install virtualenv`
+`pip install virtualenv`
```

Comment:
Use the default `venv` module which is part of Python (I think since version 3.0). So, no need to install the `virtualenv` module here and ...